src/features/navigation/RootNavigator/types.ts

```diff
@@ -69,8 +69,18 @@ export type OnboardingRootStackParamList = {
 }
 
 export type TrustedDeviceRootStackParamList = {
-  AccountSecurity: { token: string }
-  AccountSecurityBuffer: { token: string }
+  AccountSecurity: {
+    token: string
+    email: string
+    reset_password_token: string
+    expiration_timestamp: number
+  }
+  AccountSecurityBuffer: {
+    token: string
+    email: string
+    reset_password_token: string
+    expiration_timestamp: number
```

Comment:
C'est `reset_password_expiration_timestamp` le nom du paramètre. J'ai tout noté dans le ticket mais ça ne vous pas aidé visiblement